src/app/profiles/store/sagas/connection.saga.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  call,
+  delay,
+  fork,
+  put,
+  take,
+  takeLatest,
+  select
+} from 'redux-saga/effects';
+import extensionId from 'app/profiles/extensionId';
+import {
+  connect,
+  CONNECT,
+  CONNECTED,
+  disconnected
+} from 'app/store/actions/connection';
+import watchPortSaga from 'app/store/sagas/watchPort.saga';
+import { isConnecting, isConnected } from '../selectors/connection';
+import { AppAction } from 'app/actions';
+
+export function* connectSaga() {
+  try {
+    const port = yield call(browser.runtime.connect, extensionId, {
+      name: `profiles_${Date.now()}`
+    });
+    yield fork(
+      watchPortSaga,
+      ({ meta }: AppAction) =>
+        !!meta?.receiver && meta?.receiver?.id === extensionId,
```

Comment:
It should work with `meta?.receiver?.id === extensionId` …

Reply:
Yup you are right, leftover of my failed attempts to use a lower level interface....